src/benchmark_runner.cc

```diff
@@ -290,6 +291,7 @@ void BenchmarkRunner::DoOneRepetition() {
                                          has_explicit_iteration_count ||
                                          ShouldReportIterationResults(i);
 
+    b.Tearndown();
```

Comment:
This is surely `Teardown`

Reply:
What do you mean?  The `Teardown` callback is passed to the benchmark instance (`b`) so I wrap the call to it over this method . 
Alternatively, we could do `b.teardown_(....)` ?

Reply:
OOOH, I see now.  you were pointing out the typo!